conf/accumulo-testing.properties

```diff
@@ -39,12 +39,15 @@ test.ci.common.accumulo.num.tablets=20
 # Optional authorizations that if specified will be randomly selected by scanners and walkers
 # Format: a,b|a,b,c|c
 test.ci.common.auths=
+# Accumulo tserver properties to set when creating a table
+test.ci.common.accumulo.server.props=\
+tserver.compaction.major.service.cs1.planner=org.apache.accumulo.core.spi.compaction.DefaultCompactionPlanner \
+tserver.compaction.major.service.cs1.planner.opts.executors=[{"name":"small","type":"internal","maxSize":"32M","numThreads":4},{"name":"medium","type":"internal","maxSize":"128M","numThreads":2},{"name":"large","type":"external","queue":"DCQ1"}]
```

Comment:
After looking into this more, we should not include an external queue as the default case unless we also include starting the other compaction services (compactor and compaction coordinator) by default. I think the best case is to have the configuration only be internal opts and then the user can decide on if they want external compactions and then start the other services.

Reply:
agreed, if you set external but don't start the external services then some compactions will not occur.

Reply:
It would probably be beneficial to also leave a comment with either an example of opts with an external queue or a link to the website example.

Reply:
addressed this in a51ef07